src/quo/components/share/share_qr_code/view.cljs

```diff
@@ -107,32 +97,22 @@
 
 (defn- profile-bottom
   [{:keys [component-width qr-data on-text-press on-text-long-press on-share-press share-qr-type]}]
-  [:<>
-   [rn/view
-    [info-label share-qr-type]
-    [info-text
-     {:width         component-width
-      :ellipsize?    true
-      :on-press      on-text-press
-      :on-long-press on-text-long-press}
-     qr-data]]
-   [share-button
-    {:alignment :center
-     :on-press  on-share-press}]])
+  [rn/view
```

Comment:
Fragmanet `[:<>` can be used here

Reply:
It doesn't look as expected with fragment